test/e2e/mock/bl.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable camelcase */
+export const generate_bid = (interestGroup, auctionSignals, perBuyerSignals, trustedBiddingSignals, browserSignals) => {
```

Comment:
```suggestion
export const generateBid = (interestGroup, auctionSignals, perBuyerSignals, trustedBiddingSignals, browserSignals) => {
```

Reply:
the branch I am working on still expects generate_bid.